### ISSUE TITLE
Install for raspberry pi

### ISSUE THREAD

**Issue by hawkerboy7** [nwjs/npm-installer#17]
Using `npm install nw` on the RaspberryPi gives an error:
Couldn't connect to http://dl.nwjs.io/v0.12.0/nwjs-v0.12.0-linux-arm.tar.gz (404)

[Here](https://github.com/jtg-gg/node-webkit/releases) I found the package.
How can I get this to work with the npm-installer.
I was thinking about somehow redirecting the url to download the correct file.
Or is there no way for the npm-installer to support this version of linux-arm?

**Comment by shama** [comment]
Yeah we can add the platform here: https://github.com/nwjs/npm-installer/blob/master/scripts/install.js#L28

**Comment by hawkerboy7** [comment]
I tried some stuff and I think it works. Only problem is that I don't have a Pi v2 so I will not work anyways.
Maybe you could add a redirect in [http://dl.nwjs.io/v0.12.0/](http://dl.nwjs.io/v0.12.0/) so <br>`http://dl.nwjs.io/v0.12.0/nwjs-v0.12.0-linux-arm.tar.gz` will open <br>`https://github.com/jtg-gg/node-webkit/releases/nwjs-v0.12.0-linux-arm.tar.gz`.

I don't know if I am asking something ridiculus now :) but on my pi I changed the install.js file to the github url and everything appeared to be working correctly (except for me not wotking on a Pi v2)

**Comment by shama** [comment]
Hmm that is weird, maybe it's not official? I'll look into it and see if I can figure out what's going on there.

**Comment by shama** [comment]
Here is the related issue: nwjs/nw.js#1151 It looks like support is in progress and that fork is the work in progress.

**Comment by hawkerboy7** [comment]
Yes I found that one, __jtg-gg__ has apparently made a version that should work. So maybe you could make the npm-installer link to __jtg-gg's__ url containing the correct version for the Raspberry Pi?

**Comment by shama** [comment]
I think it's a work in progress though so maybe better to wait until https://github.com/nwjs/nw.js/issues/1151 lands as it looks like it will soon.

**Comment by hawkerboy7** [comment]
Ok thanks for the info!

**Comment by TomAshley303** [comment]
will nwjs or electron run on raspi 3 model b?

**Comment by Mithgol** [comment]
`npm install nw@0.21.6-sdk --nwjs_urlbase=https://github.com/LeonardLaszlo/nw.js-armv7-binaries/releases/download/v` might work to install [NW.js ARMv7 binaries](https://github.com/LeonardLaszlo/nw.js-armv7-binaries) from @LeonardLaszlo.
